src/index.js

```diff
@@ -0,0 +1,32 @@
+import './style.css';
+
+const toDoTasks = [
+  {
+    description: 'Wash the dishes',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Complete the To-Do-List project',
+    completed: false,
+    index: 1,
+  },
+];
+const listContainer = document.querySelector('.items_list');
+
+const renderList = () => {
+  let html = '';
+  toDoTasks.forEach((task, i) => {
+    html = `<div class="items">
+        <li>
+          <i class="far fa-square"></i>
+          <i class="fas fa-check-circle hidden"></i>
```

Comment:
- [ ] Kindly note that the checkbox should be added dynamically from the array of objects.

something like this
```
<input type="checkbox" ${elem.completed ? 'checked':''} name="" id="" />
```
OR
```
<input type="checkbox" ${elem.completed && 'checked'} name="" id="" />
```

depending on the logic that appears more appealing to you,

- Note that 'checked' is the attribute that checks a checkbox 😄.(checks a checkbox, lol). Just like the `required` attribute for inputs it doesn't require a value.